src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
Consider using an arrow function for `makeCalculator` to ensure that the function does not have its own `this` context, which is not needed here.